types/types.go

```diff
@@ -457,6 +466,10 @@ func GetShareManagerPodNameFromShareManagerName(smName string) string {
 	return shareManagerPrefix + smName
 }
 
+func GetConfigMapNameNameFromShareManagerName(smName string) string {
```

Comment:
Change to `GetConfigMapNameFromShareManagerName`

Reply:
Updated. Thank you.